### ISSUE TITLE
Forcing creation of ID3 tag when APE tag exists

### ISSUE THREAD

**Issue by m0nkge** [Zeugma440/atldotnet#227]
## The problem

I am trying to create ID3 2.3 tag with chapters to files with existing APE tags. When my code writes the chapters, the APE tag is updated but no ID3 tag is created. When no tag of any kind exists it creates the ID3 tag which is what I want. How can I force it to create an ID3 tag even if APE tag exists?

## Environment

* ATL version (or git revision) that exhibits the issue: 4.36
* Last ATL version that did not exhibit the issue (if applicable):
* OS/version used to run ATL: W10 22H2

## Details

Nothing further to add.

## Code To Reproduce Issue [ Good To Have ]

        theTrack.Chapters = New System.Collections.Generic.List(Of ChapterInfo)
        For Each row As DataGridViewRow In dgvChapters.Rows
            Dim ch As ATL.ChapterInfo = New ATL.ChapterInfo()
            ' stuff happens here
            theTrack.Chapters.Add(ch)
        Next
        theTrack.Save()

**Comment by Zeugma440** [comment]
Thanks for your feedback. The feature to force the creation of a given tag is missing but can be added painlessly.

I'll get back to you soon~

**Comment by Zeugma440** [comment]
`Track.Save` has a new signature 

```cs
public bool Save(MetaDataIOFactory.TagType? tagType = null, Action<float> writeProgress = null)
```

**tagType** = Tag type to save.
- Use null or TagType.ANY to save on existing tags
- Use any other TagType to save on existing tags and on the given TagType, if supported</param>


Adding an ID3v2.3 tag on a file with an APE tag should work like that :

```cs
ATL.Settings.ID3v2_tagSubVersion = 3; // Default behaviour is ID3v2.4
Track theTrack = new Track(_fileWithApe_);
theTrack.Save(MetaDataIOFactory.TagType.ID3V2);
```

Calling `Track.Save()` still behaves like before.

Are you okay with that ?

By the way, the issue with chapters is because APE... does not support chapters 😅  You should definitely use ID3v2 for that.

**Comment by Zeugma440** [comment]
PS : There's nothing published yet; I'm waiting for your opinion about the new interface

**Comment by m0nkge** [comment]
Yeah, perfect. Thank you.

**Comment by Zeugma440** [comment]
Available on today's v5.07
